lib/InstantSearch.js

```diff
@@ -7,13 +7,33 @@ var union = require('lodash/array/union');
 
 var EventEmitter = require('events').EventEmitter;
 
+var urlSyncWidget = require('./url-sync');
+
+function _enhanceConfiguration(configuration, widgetDefinition) {
```

Comment:
Would put this utility bottom so that we see first the class definition. Then there's no need to denote it _private since it's not exposed.